Navigation/Photos/PhotosTableViewCell.swift

```diff
@@ -112,16 +108,13 @@ extension PhotosTableViewCell: UICollectionViewDataSource {
     
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         let cell = photosPreview.dequeueReusableCell(withReuseIdentifier: String(describing: PhotosCollectionViewCell.self), for: indexPath) as! PhotosCollectionViewCell
```

Comment:
Слишком длинная строка, а так же force unwrap. Хоть тут и очевидно, что ячейка скастится до нужного типа, но лучше использовать безопасное разворачивание опционала. Например записать строчку в таком виде:
```suggestion
        guard let cell = collectionView.dequeueReusableCell(
            withReuseIdentifier: String(describing: PhotosCollectionViewCell.self),
            for: indexPath) as? PhotosCollectionViewCell else { return UICollectionViewCell() }
```
Способ который я предложил имеет свой недостаток - может выйти ошибка связанная с `reusIdIentifire`.
Попробуй придумать способ с использованием безопасного извлечения и без этого недостатка)